salsah/src/public/js/jquery.searchbox.js

```diff
@@ -185,6 +185,8 @@
 //						position: 'absolute',
 						left: pos.left,
 						top: pos.top +$this.outerHeight(),
+						background: 'white',
+						'z-index': 2, /* :( i don't like black magic numbers or const */
```

Comment:
CSS z-index is indeed difficult (and different every browser) ...

if it works, I guess it's fine